apiserver/facades/controller/caasunitprovisioner/provisioner.go

```diff
@@ -269,6 +271,7 @@ func (f *Facade) provisioningInfo(model Model, tagString string) (*params.Kubern
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	operatorImagePath := podcfg.GetJujuOCIImagePath(controllerCfg, version.Current)
```

Comment:
This should be consistent with caasoperator provisioner?

```
	vers, ok := modelConfig.AgentVersion()
	if !ok {
		return params.OperatorProvisioningInfo{}, errors.NewNotValid(nil,
			fmt.Sprintf("agent version is missing in model config %q", modelConfig.Name()),
		)
	}
	vers.Build = 0
	imagePath := podcfg.GetJujuOCIImagePath(cfg, vers)
```

Reply:
@wallyworld for some reason I remember AgentVersion always returning ok = false here. I'll have a look again.